Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM node:slim as build-env
```

Comment:
with dockerfile we also should be adding in commands in Makefile to build and push the dockerfile with specific tags